HLSMergerPlugin/src/test/java/io/antmedia/plugin/HLSMergerPluginTest.java

```diff
@@ -0,0 +1,214 @@
+package io.antmedia.plugin;
+
+import static org.bytedeco.ffmpeg.global.avcodec.AV_CODEC_ID_H264;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.ArgumentMatchers.any;
+import static org.mockito.ArgumentMatchers.anyLong;
+import static org.mockito.ArgumentMatchers.anyString;
+import static org.mockito.ArgumentMatchers.eq;
+import static org.mockito.ArgumentMatchers.nullable;
+import static org.mockito.Mockito.doNothing;
+import static org.mockito.Mockito.mock;
+import static org.mockito.Mockito.when;
+import static org.mockito.Mockito.spy;
+import static org.mockito.Mockito.verify;
+
+import java.io.File;
+import java.nio.file.Files;
+
+import org.junit.Before;
+import org.junit.Test;
+import org.mockito.ArgumentCaptor;
+import org.red5.server.api.scope.IScope;
+import org.springframework.context.ApplicationContext;
+
+import io.antmedia.AntMediaApplicationAdapter;
+import io.antmedia.datastore.db.types.Broadcast;
+import io.antmedia.muxer.HLSMuxer;
+import io.antmedia.muxer.IAntMediaStreamHandler;
+import io.antmedia.muxer.MuxAdaptor;
+import io.antmedia.muxer.Muxer;
+import io.vertx.core.Vertx;
+
+
+public class HLSMergerPluginTest {
+
+    private HLSMergerPlugin plugin;
+    private ApplicationContext context;
+    private Vertx vertx;
+    private IAntMediaStreamHandler streamHandler;
+    private IScope scope;
+
+    @Before
+    public void setup() {
```

Comment:
Please add integration test code as well. Ant Media Server is installed in the pipeline and sample integration test is available for Media Push plugin